From c2da4903ec29f0af0821f7b17e7a4b6d52631d03 Mon Sep 17 00:00:00 2001
From: Martin Bouchet <tincho5588@gmail.com>
Date: Tue, 7 Mar 2017 13:15:59 -0300
Subject: [PATCH 091/173] Snap: grant android.permission.RECEIVE_BOOT_COMPLETED
 permisions

Log:
03-07 13:00:45.697  2812  2830 W BroadcastQueue: Permission Denial:
receiving Intent { act=android.intent.action.BOOT_COMPLETED
flg=0x9000010 (has extras) } to
org.cyanogenmod.snap/com.android.camera.DisableCameraReceiver requires
android.permission.RECEIVE_BOOT_COMPLETED due to sender null (uid 1000)

Change-Id: I55cd1c96ac1a06b644fc207c235f694c8a60da99
---
 AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index acb5abdcf..fbb0d756f 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -15,6 +15,7 @@
     <uses-permission android:name="android.permission.CAMERA" />
     <uses-permission android:name="android.permission.NFC" />
     <uses-permission android:name="android.permission.READ_SYNC_SETTINGS" />
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
     <uses-permission android:name="android.permission.VIBRATE" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
-- 
2.17.1

